pantheon-bundle/src/main/resources/SLING-INF/nodetypes/nodetypes.cnd

```diff
@@ -84,6 +84,9 @@
 [pant:assemblyVersion] > nt:unstructured, sling:Resource, mix:referenceable, mix:created, mix:lastModified
     - sling:resourceType (string) = 'pantheon/assemblyVersion' mandatory autocreated
 
+[pant:assemblyPage] > nt:unstructured, sling:Resource, mix:referenceable, mix:created, mix:lastModified
```

Comment:
what's the purpose of having pant:assemblyPage?

Reply:
As you know, long-term we are going to want to break up large assemblies. For now, we've agreed the assemblies are small enough to not need it, but the thought is to future-proof the data structure. So all of our assemblies today will just have "one page" (the entire assembly). But it will be stored in such a way that we can have future multi-page assemblies and our data structure will not need to change to accommodate it.

Does that answer the question or were you trying to ask something else?